### PR TITLE
Mtc docs additional changes

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
### Checklist

- [x] Appropriate branch selected => This docs PR is against the `mtc-docs` branch.
- [na] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [na] The description lists all applicable issues this PR seeks to resolve
- [na] The description lists any configuration setting(s) that differ from the default settings
- [na] All tests and CI builds passing

### Description

Includes changes requested by client:
- new/updated screenshots
- Changing merge rules sublists to roman (not natively supported in markdown).
- Some tweaks in the merge rules text.

### CODE REVIEW

File: docs/user/merging-feeds.md [hunk 1]
```diff
@@ -25,37 +25,42 @@ There are a set of rules that govern the requirements for input feed versions an
...
-       that are either stations, entrances/exits, or generic nodes (e.g., for
-       `pathways.txt`). The merge will happen on `stop_code` if provided, or fallback on stop_id.
-    2. For regular stops (`location_type = 0` or empty), all or none of the stops must
-       contain `stop_codes`. Otherwise, the merge feeds job will be failed.
+
+    (i) Stops with `location_type` greater than `0` (i.e., anything but `0` or `empty`) are permitted
+    to have empty `stop_codes` (even if there are other stops in the feed that have
+    `stop_code` values). This is because these location_types represent special entries
+    that are either stations, entrances/exits, or generic nodes (e.g., for
+    `pathways.txt`). The merge will happen on `stop_code` if provided, or fallback on stop_id.
```
Comment (daniel-heppner-ibigroup) [inline docs/user/merging-feeds.md:33]: add code block for `stop_id`

File: docs/user/merging-feeds.md [hunk 2]
```diff
@@ -25,37 +25,42 @@ There are a set of rules that govern the requirements for input feed versions an
...
-       to have empty `stop_codes` (even if there are other stops in the feed that have
-       `stop_code` values). This is because these location_types represent special entries
-       that are either stations, entrances/exits, or generic nodes (e.g., for
-       `pathways.txt`). The merge will happen on `stop_code` if provided, or fallback on stop_id.
-    2. For regular stops (`location_type = 0` or empty), all or none of the stops must
-       contain `stop_codes`. Otherwise, the merge feeds job will be failed.
+
+    (i) Stops with `location_type` greater than `0` (i.e., anything but `0` or `empty`) are permitted
+    to have empty `stop_codes` (even if there are other stops in the feed that have
+    `stop_code` values). This is because these location_types represent special entries
```
Comment (daniel-heppner-ibigroup) [inline docs/user/merging-feeds.md:31]: add code block for `location_type`s

**Review by daniel-heppner-ibigroup — APPROVED** [review]
LGTM with minor comments

File: docs/user/merging-feeds.md (hunk 1 above)
Comment (binh-dam-ibigroup) [reply docs/user/merging-feeds.md]: Thanks, good catch, fixed in https://github.com/ibi-group/datatools-ui/pull/757/commits/caec4a1099e14344383610120a94992eb4b1191d.

File: docs/user/merging-feeds.md (hunk 2 above)
Comment (binh-dam-ibigroup) [reply docs/user/merging-feeds.md]: Ditto fixed in https://github.com/ibi-group/datatools-ui/pull/757/commits/caec4a1099e14344383610120a94992eb4b1191d

**Review by miles-grant-ibigroup — APPROVED** [review]
I would double check that the screenshots are rendering in the deployed docs -- they're not rendering in the GitHub preview. Otherwise looks great!

**Pull request merged**